code-challenges.js

```diff
@@ -47,3 +96,16 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+//This function establishes two parameters for arguments to be passed in later
+const numberSet = (num1, num2) => {
+//This if statement specifies that if the first parameter is greater than the second parameter, then the statement will return the difference of the first and second parameters
+  if(num1 > num2){
+    return num1 - num2
+//This else statement returns any other parameter entry, which for the purposes of this problem is really just num1 < num2
+  }else {
+    return num2 - num1
+  }
+}
+//This command passes in both varieties of arguments in order to receive the expected output
+console.log(numberSet(number1, number2), numberSet(number3, number4))
```

Comment:
Excellent!  Cool approach to running both sets in the same console log and great job with keeping the function dynamic!  If you want to stretch yourself, can you think of how this would handle a different data type being passed in such as a string?